src/include/drbd/helps.rb

```diff
@@ -134,7 +134,13 @@ def initialize_drbd_helps(include_target)
           "<p><b><big>Global Configuration of DRBD</big></b></p>"
         ) +
           _(
-            "<p>Check <b>\"Udev Always Use VNR\"</b> treat implicit the same as explicit volumes, only one single volume with the implicit volume number 0</p>"
+            "<p>Check <b>\"Udev Always Use VNR\"</b> treat implicit define volume number the same\n" +
+              "                as explicit volumes. When udev asks drbdadm for a list of device\n" +
```

Comment:
IIRC the help text is treated similarly to the HTML format, i.e. multiple spaces are reduced to one, the new lines are treated as spaces. So I think you can remove the "\n" and just keep a single space.

---

BTW the `+` operator is not needed in Ruby, you can use `\` for splitting a string over several lines, like:

```ruby
_("this is " \
  "a very " \
  "long text")
```

The Ruby gettext handles that well.

This is a bit more effective, with `+` Ruby creates the string parts objects first and then concatenates them to a new object and deletes the parts objects not needed anymore. With `\` it creates the joined string directly without the temporary objects so it's a bit more effective.

Reply:
Thanks:) Will fix it soon.